### PR TITLE
Prepare release version 2.0.6

### PR DESCRIPTION
**Pull request by kawsarahmedr**
=== Changelog ===

1. Update the Key model and include the (order item id) prop

### CODE REVIEW

File: includes/Admin/Orders.php [hunk 1]
```diff
@@ -214,10 +214,15 @@ public static function display_order_item_meta( $item_id, $item, $product ) {
 			return;
 		}
 		$keys = wcsn_get_keys(
-			array(
-				'order_id'   => $order_id,
-				'product_id' => $product->get_id(),
-				'limit'      => - 1,
+			apply_filters(
+				'wc_serial_numbers_admin_order_item_keys_query_args',
```
Comment (sultann) [inline includes/Admin/Orders.php:218]:
```suggestion
				'wcsn_order_item_keys_query_args',
```

File: src/functions.php [hunk 2]
```diff
@@ -490,12 +490,18 @@ function wcsn_order_update_keys( $order_id ) {
 			}
 
 			$delivered_qty = Key::count(
-				array(
-					'order_id'       => $order_id,
-					'product_id'     => $item['product_id'],
-					'status__not_in' => array( 'cancelled' ),
+				apply_filters(
+					'wc_serial_numbers_order_item_keys_count_query_args',
```
Comment (sultann) [inline src/functions.php:494]:
```suggestion
				'wcsn_order_item_keys_count_args',
```

**Review by sultann — CHANGES_REQUESTED** [review]

File: includes/Admin/Orders.php (hunk 1 above)
Comment (kawsarahmedr) [reply includes/Admin/Orders.php]: Updated!

File: src/functions.php (hunk 2 above)
Comment (kawsarahmedr) [reply src/functions.php]: Updated!

**Review by sultann — APPROVED** [review]

**Pull request merged**